assets/js/theme/checkout/cvv-mask.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import MaskedInput from 'react-text-mask';
+
+export default class CvvMaskComponent extends React.Component {
```

Comment:
This component can just be a stateless functional component, as you only have a `render` method.